<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsOtherPeripherals</class>
 <widget class="QWidget" name="SettingsOtherPeripherals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutRtc">
     <item>
      <widget class="QLabel" name="labelRtc">
       <property name="text">
        <string>ISA RTC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxRTC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxVisibleItems">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConfigureRTC">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMem">
     <property name="title">
      <string>ISA Memory Expansion</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutMem">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIsaMemCard1">
        <property name="text">
         <string>Card 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxIsaMemCard1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaMemCard1">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelIsaMemCard2">
        <property name="text">
         <string>Card 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxIsaMemCard2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaMemCard2">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIsaMemCard3">
        <property name="text">
         <string>Card 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxIsaMemCard3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaMemCard3">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelIsaMemCard4">
        <property name="text">
         <string>Card 4:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxIsaMemCard4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaMemCard4">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxRom">
     <property name="title">
      <string>ISA ROM Cards</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutRom">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIsaRomCard1">
        <property name="text">
         <string>Card 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxIsaRomCard1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaRomCard1">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelIsaRomCard2">
        <property name="text">
         <string>Card 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxIsaRomCard2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaRomCard2">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIsaRomCard3">
        <property name="text">
         <string>Card 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxIsaRomCard3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaRomCard3">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelIsaRomCard4">
        <property name="text">
         <string>Card 4:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxIsaRomCard4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonConfigureIsaRomCard4">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutIBPC">
     <item>
      <widget class="QCheckBox" name="checkBoxISABugger">
       <property name="text">
        <string>ISABugger device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxPOSTCard">
       <property name="text">
        <string>POST card</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUT">
     <item>
      <widget class="QCheckBox" name="checkBoxUnitTester">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>86Box Unit Tester</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConfigureUT">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutKeyCard">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxKeyCard">
       <property name="text">
        <string>Novell NetWare 2.x Key Card</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConfigureKeyCard">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxRTC</tabstop>
  <tabstop>pushButtonConfigureRTC</tabstop>
  <tabstop>comboBoxIsaMemCard1</tabstop>
  <tabstop>pushButtonConfigureIsaMemCard1</tabstop>
  <tabstop>comboBoxIsaMemCard2</tabstop>
  <tabstop>pushButtonConfigureIsaMemCard2</tabstop>
  <tabstop>comboBoxIsaMemCard3</tabstop>
  <tabstop>pushButtonConfigureIsaMemCard3</tabstop>
  <tabstop>comboBoxIsaMemCard4</tabstop>
  <tabstop>pushButtonConfigureIsaMemCard4</tabstop>
  <tabstop>comboBoxIsaRomCard1</tabstop>
  <tabstop>pushButtonConfigureIsaRomCard1</tabstop>
  <tabstop>comboBoxIsaRomCard2</tabstop>
  <tabstop>pushButtonConfigureIsaRomCard2</tabstop>
  <tabstop>comboBoxIsaRomCard3</tabstop>
  <tabstop>pushButtonConfigureIsaRomCard3</tabstop>
  <tabstop>comboBoxIsaRomCard4</tabstop>
  <tabstop>pushButtonConfigureIsaRomCard4</tabstop>
  <tabstop>checkBoxISABugger</tabstop>
  <tabstop>checkBoxPOSTCard</tabstop>
  <tabstop>checkBoxUnitTester</tabstop>
  <tabstop>pushButtonConfigureUT</tabstop>
  <tabstop>checkBoxKeyCard</tabstop>
  <tabstop>pushButtonConfigureKeyCard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
